pop-control.lua

```diff
@@ -0,0 +1,127 @@
+-- Script to control the various population caps as well as use of max-wave and hermit persistently per fortress
+-- by Tachytaenius
+--[====[
+pop-control
+===========
+Controls hermit and the various population caps per-fortress.
+Not safe with gui/settings-manager as it will commit temporary
+population cap changes permanently to file.
+Intended to be placed within ``onMapLoad.init`` as ``pop-control on-load``.
+Available arguments:
+
```

Comment:
shouldn't there be a way to initialize this script with the settings that you want?

Reply:
Like, apply defaults and skip the GUI? OK, makes sense.